.bazelrc

```diff
@@ -150,10 +150,12 @@ build:engflowbase --config=lintonbuild
 test:engflowbase --test_env=REMOTE_EXEC=1
 test:engflowbase --test_env=GOTRACEBACK=all
 build:engflow --config=engflowbase
-build:engflow --remote_cache=grpcs://tanzanite.cluster.engflow.com
-build:engflow --remote_executor=grpcs://tanzanite.cluster.engflow.com
-build:engflow --bes_backend=grpcs://tanzanite.cluster.engflow.com
-build:engflow --bes_results_url=https://tanzanite.cluster.engflow.com/invocation/
+build:engflow --remote_cache=grpcs://mesolite.cluster.engflow.com
```

Comment:
You can refactor this by moving the `--remote_cache`, `--remote_executor`, `--bes_backend`, and `--bes_results_url` configurations (for both `engflow` and `engflowpublic`) under `build:engflowbase`.

Reply:
Indeed. Right. Done with caveate: `--bes_results_url` is different for the two, so I didn't move it to `build:engflowbase`.